addons/hr_holidays_attendance/tests/test_holidays_overtime.py

```diff
@@ -243,7 +260,28 @@ def test_public_leave_overtime(self):
         })
         self.assertNotEqual(leave.company_id, self.employee.company_id)
```

Comment:
it could be nice to make the company explicit in the leave creation if it has a significance rather than just checking it's ok